client/README.md

```diff
@@ -10,7 +10,7 @@
 ### Relevant Documentation
 
 * [PNPM](https://pnpm.io/)
-* [uvu](https://github.com/lukeed/uvu)
+* [Mocha](https://mochajs.org)
```

Comment:
Curious, what was wrong with uvu? I don't understand the context for a lot of JS tool changes so it's not obvious to me.

Reply:
Nothing, really. The reason Mocha is used is because the test runner uses it by default, and is well integrated with it. It also has some advantages when being run in a browser, I think.